cmd/internal/backup/providers/gcp/gcp.go

```diff
@@ -160,40 +160,24 @@ func (b *BackupProviderGCP) DownloadBackup(ctx context.Context, version *provide
 		downloadFileName = filepath.Base(downloadFileName)
 	}
 
-	backupFilePath := filepath.Join(outDir, downloadFileName)
-
-	b.log.Info("downloading", "object", version.Name, "gen", gen, "to", backupFilePath)
```

Comment:
I think the log line is still useful, but we can just omit the `to` field. What do you think?